src/structs.jl

```diff
@@ -420,7 +425,7 @@ Base.@kwdef struct Model
     constraints::Vector{Constraint} = Constraint[]
     reactions::Dict{String,Reaction} = Dict()
     objectives::Dict{String,Objective} = Dict()
-    active_objective::String = ""
+    active_objective::Maybe{String} = nothing
```

Comment:
Note that I originally [had this as `Maybe{String}`](https://github.com/LCSB-BioCore/SBML.jl/commit/a3461be3d566e1696ddc08f45115b5a85ea72b7f), but then [changed it to `String`](https://github.com/LCSB-BioCore/SBML.jl/commit/f385a0da62e4f05de7cba9255cd79f4cc2579903) because libsbml treats the active objective ID as an empty string, instead of `NULL`, when there is no active objective.  I think I had problems with the roundtrip checks because of this.

Reply:
yeah the `FbcModelPlugin_isSetActiveObjective` is missing in the C API, so now I check the string emptiness. Otherwise it's the same thing as with the other emptiable strings. I'll eventually make a PR fix to libsbml.

Reply:
Ok, sounds good!